tools/warning-to-plist/tests/unit/clang_sanitizer_output_test_files/files/ubsan1.cpp

```diff
@@ -0,0 +1,6 @@
+int main(int argc, char **argv)
+{
+  int k = 0x7fffffff;
+  k += argc;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'k' is assigned a value that is never used.](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4169881)